install.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash -e
 
-# Installing pip-10.0.1
+export PYTHON_PIP=18.0
```

Comment:
The heap said no update for PIP. Please make sure the heap is documented correctly.